glusterd2/volume/volume-utils.go

```diff
@@ -199,3 +200,54 @@ func CreateVolumeInfoResp(v *Volinfo) *api.VolumeInfo {
 		SnapList:  v.SnapList,
 	}
 }
+
+// MountLocalBricks mounts bricks of auto provisioned volumes
+func MountLocalBricks() error {
```

Comment:
Can we have this function in common utils, so that the same can be used for snapshot also. Otherwise I have to use another function during glusterd2 init.

Reply:
Since it is not generic function it depends on store utils, can't move to common utils. Let me see

Reply:
You could create a glusterd2/mountutils or just glusterd2/utils instead of pkg/utils